dicom-viewer.yml

```
@@ -0,0 +1,15 @@
+---
+services:
+  ocis:
+    depends_on:
+      dicomviewer-init:
+        condition: service_completed_successfully
+
+  dicomviewer-init:
+    image: owncloud/web-app-dicom-viewer/dicom-viewer-1.0.0
+    user: root
+    volumes:
+      - ocis-apps:/apps
+    entrypoint:
+      - /bin/sh
+    command: ["-c", "cp -R /var/lib/nginx/html/dicom-viewer/ /apps"]
```

```suggestion
    command: ["-c", "cp -R /app/js/* /apps"]
```
I haven't used nginx but simple node alpine to create image. I don't think this make any differences

---

path depends on how it is done in the docker image